R/use_extendr.R

```diff
@@ -265,6 +283,10 @@ use_rextendr_template <- function(template,
     overwrite = overwrite
   )
 
+  if (isTRUE(as_executable)) {
+    Sys.chmod(save_as, mode = "755")
```

Comment:
I do not know if this will work on Windows.

Reply:
It seems working on CI

Reply:
It would be good to have a windows user confirm this. Given that it is part of `{base}` I think it ought to work. How it works may be different, though?

> It may not be supported on a system (when a warning is issued).

So long as there is no warning emitted in CI, I think we can confidently say it works on windows.